test_scripts/MobileProjection/Phase2/001_audioStreamingState_001-009.lua

```diff
@@ -0,0 +1,46 @@
+---------------------------------------------------------------------------------------------------
+-- Issue:
```

Comment:
General comment : Propose to split scripts to make set more flexible.

Reply:
@GetmanetsIrina For now it was agreed to leave current approach